### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by pgherveou**

### CODE REVIEW

File: ethbloom/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.14.0] - 2024-09-10
```
Comment (ordian) [inline ethbloom/CHANGELOG.md:9]:
```suggestion
## [0.14.0] - 2024-09-11
```

File: ethereum-types/CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
-### Breaking
+
+## [0.15.0] - 2024-09-10
```
Comment (ordian) [inline ethereum-types/CHANGELOG.md:9]:
```suggestion
## [0.15.0] - 2024-09-11
```

File: keccak-hash/CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
-## [0.12.0] - 2022-09-20
+## [0.11.0] - 2024-09-10
```
Comment (ordian) [inline keccak-hash/CHANGELOG.md:9]:
```suggestion
## [0.11.0] - 2024-09-11
```

File: primitive-types/CHANGELOG.md [hunk 4]
```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
-### Breaking
+
+## [0.13.0] - 2024-09-10
```
Comment (ordian) [inline primitive-types/CHANGELOG.md:9]:
```suggestion
## [0.13.0] - 2024-09-11
```

File: primitive-types/impls/num-traits/CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.2.0] - 2024-09-10
```
Comment (ordian) [inline primitive-types/impls/num-traits/CHANGELOG.md:9]:
```suggestion
## [0.2.0] - 2024-09-11
```

File: primitive-types/impls/rlp/CHANGELOG.md [hunk 6]
```diff
@@ -4,7 +4,8 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
-## [Unreleased]
+## [0.4.0] - 2024-09-10
```
Comment (ordian) [inline primitive-types/impls/rlp/CHANGELOG.md:7]:
```suggestion
## [0.4.0] - 2024-09-11
```

File: primitive-types/impls/serde/CHANGELOG.md [hunk 7]
```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
-### Breaking
+
+## [0.5.0] - 2024-09-10
```
Comment (ordian) [inline primitive-types/impls/serde/CHANGELOG.md:9]:
```suggestion
## [0.5.0] - 2024-09-11
```

File: rlp-derive/CHANGELOG.md [hunk 8]
```diff
@@ -4,7 +4,8 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
-## [Unreleased]
+## [0.2.0] - 2024-09-10
```
Comment (ordian) [inline rlp-derive/CHANGELOG.md:7]:
```suggestion
## [0.2.0] - 2024-09-11
```

File: uint/CHANGELOG.md [hunk 9]
```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
-### Breaking
+
+## [0.10.0] - 2024-09-10
```
Comment (ordian) [inline uint/CHANGELOG.md:9]:
```suggestion
## [0.10.0] - 2024-09-11
```

File: rlp/CHANGELOG.md [hunk 10]
```diff
@@ -4,8 +4,9 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
-## [Unreleased]
+## [0.6.0] - 2024-09-10
```
Comment (ordian) [inline rlp/CHANGELOG.md:7]:
```suggestion
## [0.6.0] - 2024-09-11
```

File: rlp/CHANGELOG.md [hunk 11]
```diff
@@ -4,8 +4,9 @@ The format is based on [Keep a Changelog].
...
 - Migrated to 2021 edition, enforcing MSRV of `1.56.1`. [#601](https://github.com/paritytech/parity-common/pull/601)
+- Updated `primitive-types` to 0.13. [#859](https://github.com/paritytech/parity-common/pull/859)
```
Comment (ordian) [inline rlp/CHANGELOG.md:9]: this is a dev dependency, same as above
```suggestion
```

File: rlp-derive/CHANGELOG.md [hunk 12]
```diff
@@ -4,7 +4,8 @@ The format is based on [Keep a Changelog].
...
+- Updated `rlp` to 0.6. [#859](https://github.com/paritytech/parity-common/pull/859)
```
Comment (ordian) [inline rlp-derive/CHANGELOG.md:8]: this is a dev-dependency, so it shouldn't affect versioning, but there was another breaking unreleased change
```suggestion
```

**Review by ordian — APPROVED** [review]
Sorry I missed notification, so need to readjust the publish dates.

**Comment by ordian** [comment]
Will publish these

**Comment by ordian** [comment]
Published

**Comment by pgherveou** [comment]
> Published

Thanks @ordian  for doing this 🙏

**Pull request merged**